src/app/modules/sgid/sgid.service.ts

```diff
@@ -212,17 +224,79 @@ export class SgidServiceClass {
   }
 
   /**
-   * Verifies a sgID JWT and extracts its payload.
+   * Create a JWT based with access token from sgID
```

Comment:
Would be helpful to add a short liner about how this is different from createSgidSingpassJwt, for future ref

E.g. something like
* Create a JWT based with access token from sgID
* This access token is then used to exchange for myinfo scopes in the public form controller
* Unlike `createSgidSingpassJwt` where the access token is exchanged for userInfo upfront and the userInfo is stored in the jwt